tests/functional/connect_another_device.js

```diff
@@ -179,16 +175,17 @@ define([
       const forceUA = UA_STRINGS['desktop_firefox'];
 
       return this.remote
-        // preVerified: false causes the "verify account" email to be sent,
-        // that's used later to verify.
-        .then(createUser(signUpEmail, PASSWORD, { preVerified: false }))
-        .then(createUser(signInEmail, PASSWORD, { preVerified: true }))
+      // preVerified: false causes the "verify account" email to be sent,
+      // that's used later to verify.
+        .then(createUser(signUpEmail, PASSWORD, {preVerified: false}))
+        .then(createUser(signInEmail, PASSWORD, {preVerified: true}))
```

Comment:
I thought these made more sense when they were indented to the same level as the code they're commenting.